dags/airflow_utils.py

```diff
@@ -103,7 +102,7 @@ def partitions(from_date: date, to_date: date, partition: str) -> List[dict]:
 # Default settings for all DAGs
 pod_defaults = dict(
     get_logs=True,
-    image_pull_policy="IfNotPresent",
+    image_pull_policy="Always",
```

Comment:
can we do something like this? only pull when a new version is available.
https://gist.github.com/byrnedo/9b2078c191360c681f85cebb2187d66f/revisions

Reply:
Adding a bit more on my thinking, I'd expect that as soon as we finish with the current cleaning, changes on the python code would become slower. Then we can introduce some kind of versioning and use semver tags. As soon as this is in place, we can move from the current solution to the second approach.

Reply:
understood, we can do something like a mix of hack and SHA hash. 
Looks good for now.